hack/e2e.sh

```diff
@@ -77,6 +78,11 @@ if [ -n "${IMAGE_FORMAT:-}" ]; then
     echo "\$IMAGE_FORMAT=$IMAGE_FORMAT"
 fi
 
+if [[ $TEST_ENABLE_MACHINE_AUTOSCALING == "true" ]]; then
```

Comment:
We would update our release CI configuration to explicitly set this value to true:

```bash
$ make e2e TEST_ENABLE_MACHINE_AUTOSCALE="true" ...
```